services/QuillLMS/app/assets/stylesheets/pages/progress_reports/diagnostic_results_summary.scss

```diff
@@ -352,8 +352,10 @@
         max-height: 100vh;
         overflow: scroll;
         header {
+          width: 512px;
```

Comment:
Just wondering if we want to make this a max-width (with a width of 100%) so it's more responsive?

Reply:
for sure! added that to the PR I just put out